### PR TITLE
Kubernetes Enterprise Operator Release 1.14.0

### PR DESCRIPTION
**Pull request by mms-build-account**
Kubernetes Enterprise Operator Release 1.14.0

### CODE REVIEW

**Review by chatton — APPROVED** [review]
LGTM!

File: dockerfiles/mongodb-enterprise-database/2.0.0/ubuntu/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ARG imagebase
 FROM ${imagebase} as base
 
-FROM ubuntu:xenial-20210416
+FROM ubuntu:xenial-20210114
```
Comment (bznein) [inline dockerfiles/mongodb-enterprise-database/2.0.0/ubuntu/Dockerfile:4]: @rodrigovalin I think we want to revert this

File: dockerfiles/mongodb-enterprise-operator/1.10.0/ubuntu/Dockerfile [hunk 2]
```diff
@@ -5,7 +5,8 @@
 ARG imagebase
 FROM ${imagebase} as base
 
-FROM ubuntu:xenial-20210416
+FROM ubuntu:xenial-20210114
```
Comment (bznein) [inline dockerfiles/mongodb-enterprise-operator/1.10.0/ubuntu/Dockerfile:8]: @rodrigovalin I think we want to revert this

File: dockerfiles/mongodb-enterprise-appdb/10.2.15.5958-1_4.2.11-ent/ubuntu/Dockerfile [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ARG imagebase
 FROM ${imagebase} as base
 
-FROM ubuntu:xenial-20210416
+FROM ubuntu:xenial-20210114
```
Comment (bznein) [inline dockerfiles/mongodb-enterprise-appdb/10.2.15.5958-1_4.2.11-ent/ubuntu/Dockerfile:4]: @rodrigovalin I think we want to revert this

**Review by bznein — COMMENTED** [review]
Looks good but I think a few changes shouldn't be there

**Review by bznein — APPROVED** [review]

**Comment by rodrigovalin** [comment]
@bznein rebased and droped the broken commits. Todo as of this, update those files.

**Pull request merged**